lib/datadog/tracing/metadata/ext.rb

```diff
@@ -47,6 +47,10 @@ module AppTypes
         module Distributed
           TAG_ORIGIN = '_dd.origin'
           TAG_SAMPLING_PRIORITY = '_sampling_priority_v1'
+
+          # Trace tags with this prefix will propagate from a trace through distributed tracing.
+          # Distributed headers tags with this prefix will be injected into the active trace.
+          TAGS_PREFIX = '_dd.p.'
```

Comment:
Borrowing from the follow up PR, which also uses this constant: https://github.com/DataDog/dd-trace-rb/pull/2266/files#diff-63ed19471af27cc17763b8dbd88e4c5d1369b4687df23d446e8a6990f6815870R52